src/Controller/UserController.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+
+namespace App\Controller;
+
+use App\Entity\UpdateUserRequest;
+use App\Form\UserType;
+use App\Services\FileUploader;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+use App\Services\UserService;
+
+class UserController extends AbstractController
+{
+
+    /**
+     * @Route("/flatmate/{uuid}", name="profile.view", methods={"GET"})
+     * @return Response
+     */
+    public function showUserProfile(UserService $userService, $uuid): Response
+    {
+        $user = $userService->getUserByUUID($uuid);
```

Comment:
o `$this->getUser()` neveikia? nes parentineja controllerio klasese yra sitas metodas(tiksliau ControllerTrait'e)

Reply:
`$this->getUser()` man grąžintų prisijungusio user objektą, bet čia šiuo atveju reikalinga gauti ne prisijungusio userio objektą, o userio, kurio `uuid` url'e įvestas

Reply:
pala pala, o tai ne prisijungusio uzerio ir reikia? kam tas uuid dar?

Reply:
yra du route'ai: /flatmate/{uuid}/`` ir `/dashboard/profile/`
/flatmate/{uuid}/`` - renderina bet kurio userio profilį
`/dashboard/profile/` - renderina prisijungusio userio editinimo formą